src/sonic-yang-models/yang-models/sonic-types.yang

```diff
@@ -107,4 +111,42 @@ module sonic-types {
             pattern "percentage|used|free|PERCENTAGE|USED|FREE";
         }
     }
+
+    typedef oper-status {
+        type enumeration {
+            enum up;
+            enum down;
+        }
+    }
+
+    typedef mode-enable {
```

Comment:
what is the mode-enable?

Reply:
This type can be used where toggle functionality required. For ex. IPv6-link-local-only, Dhcp-replay-link-select, SNMP traps etc